zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -409,12 +409,12 @@ ascan.custom.warn.disabled	= Input vectors disabled via the Custom Vectors tab
 
 ascan.desc                     = Active scanner, heavily based on the original Paros active scanner, but with additional tests added
 
-ascan.options.anticsrf.label   = Handle anti-CSRF tokens (experimental functionality)
+ascan.options.anticsrf.label   = Handle Anti-CSRF Tokens (Experimental Functionality)
 ascan.options.apply.label		= Apply
-ascan.options.attackOnStart.label = Allow Attack mode to be enabled on startup
-ascan.options.attackPrompt.label = In Attack mode prompt to rescan nodes when scope changed
-ascan.options.attackRescan.label = In Attack mode always rescan nodes when scope changed
-ascan.options.delayInMs.label  = Delay when scanning in milliseconds:
+ascan.options.attackOnStart.label = Allow Attack Mode to Be Enabled on Startup
+ascan.options.attackPrompt.label = In Attack Mode Prompt to Rescan Nodes When Scope Changed
+ascan.options.attackRescan.label = In Attack Mode Always Rescan Nodes When Scope Changed
+ascan.options.delayInMs.label  = Delay When Scanning in Milliseconds:
```

Comment:
I think a period works for these four too. You could leave the caps on "Attack Mode" since that's a name.

Reply:
For the delay one maybe: `Delay When Scanning (In Milliseconds):` because that seems to fit better with the style of the other two max settings?

Reply:
These changes will be in the next commit